packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -122,7 +123,14 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
-        Analytics.setAttributes({ account_type: loginid.substring(0, 2) });
+        Analytics.setAttributes({
+            account_type: LocalStore?.get('active_loginid')?.substring(0, 2) ?? 'unlogged',
+            app_id: getAppId(),
+            device_type: isDesktopOs() ? 'desktop' : 'mobile',
+            device_language: navigator?.language || 'en-EN',
+            user_language: getLanguage().toLowerCase(),
+            country: Cookies.get('clients_country') || Cookies.getJSON('website_status'),
```

Comment:
Don't we have 1 concrete field we source country from?

Reply:
I set this field like that to be sure that country field will not undefined. It's really important. `Cookies.getJSON('website_status')` returning proper value, but in deriv-com `Cookies.get('clients_country')` sometimes undefined